lib/src/test/java/com/auth0/jwt/JWTVerifierTest.java

```diff
@@ -453,10 +418,10 @@ public void shouldThrowOnInvalidCustomClaimValue() {
         exception.expect(InvalidClaimException.class);
         exception.expectMessage("The Claim 'name' value doesn't match the required one.");
         String token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjpbInNvbWV0aGluZyJdfQ.3ENLez6tU_fG0SVFrGmISltZPiXLSHaz_dyn-XFTEGQ";
-        Map<String, Object> map = new HashMap<>();
-        map.put("name", new Object());
-        JWTVerifier verifier = new JWTVerifier(Algorithm.HMAC256("secret"), map, Clock.systemUTC());
-        verifier.verify(token);
+        JWTVerifier.init(Algorithm.HMAC256("secret"))
+                .withClaim("name", "")
```

Comment:
Why this parameter was changed from Object to empty string?

Reply:
We don't have a constructor that accepts an object. This test is to ensure that a claim in a given name is not present and not about the value, I decided to go ahead with an empty string, I will add some value instead of an empty string here.